components/dashboard/src/components/ConfirmationModal.tsx

```diff
@@ -87,7 +54,24 @@ export default function ConfirmationModal(props: {
                 return true;
             }}
         >
-            {children}
+            <p className="mb-3 text-base text-gray-500">{props.areYouSureText}</p>
```

Comment:
Fixing an error w/ children not having unique keys - was able to simplify how we render some of this content a bit and avoid that error condition.